### ISSUE TITLE
System safari is not redirecting back to native iOS app after Email Verification

### ISSUE THREAD

**Issue by punithbm** [openid/AppAuth-iOS#582]
**Describe the bug**
We are building a native ios app using the AppAuth library. In this flow. Once the user registration is done, We need to do email verification. So the user will be verifying his/her email by going outside of the app and launching the system safari browser in iOS device which is received in the inbox and click the verification link. 
But once the verification is done successfully user is not navigated back to the application.

**To Reproduce**
1. Register
2. Keep Email Verification Mandate in Keycloak Configuration. User will be requested to verify his/her Email address after registration
3. Open the received email verification link in the iOS device safari browser. 
4. Verifies successfully and the user is not redirected back to the application.

Please let me know if you need any more information.

**Comment by punithbm** [comment]
@WilliamDenniss  Please check on this

**Comment by KevinGiacomino** [comment]
@punithbm Hello do you have resolve this issue now ?

**Comment by Nooba** [comment]
Currently facing this exact issue. Still looking for an answer.

**Comment by mdmathias** [comment]
I don't think this has to do with AppAuth-iOS, and instead sits somewhere in between some technology called Keycloak, a user's email, and/or another issue related to deep links.

I'm closing this as unrelated to AppAuth-iOS. If there are specific reproduction steps that we can take to debug a concrete example, then please reopen this issue with those steps.
